contentcuration/contentcuration/static/js/edit_channel/uploader/hbtemplates/edit_metadata_editor.handlebars

```diff
@@ -11,30 +14,57 @@
 		<h4>Title<label class="required">*<span id="title_error">Title cannot be blank.</span></label></h4>
 		<input type="text" id="input_title" class="upload_input input_listener" placeholder="Enter a title" maxlength="200" value="{{node.title}}" required/>
 	{{/if}}
-	<h4>Author</h4>
-	<input type="text" id="author_field" class="upload_input input_listener" value="{{author}}" placeholder="Enter author name" disabled/>
-	{{#if is_file}}
-	<div class="row">
-		<div class="content_nodes_only col-sm-5">
-			<h4>License</h4>
-			<select id="license_select">
-	            <option selected disabled value=0>-</option>
-	            {{#each licenses}}
-	                <option value={{id}}>{{license_name}}</option>
-	            {{/each}}
-	        </select>
-	        <span class="glyphicon glyphicon glyphicon-info-sign" aria-hidden="true" id="license_about" ></span>
+	{{#if isoriginal}}
```

Comment:
https://trello.com/c/60LZzI3r/192-when-importing-content-item-across-channels-some-metadata-should-be-locked-and-no-longer-be-editable-e-g-license-copyright-holde